pkg/cache/multi_namespace_cache.go

```diff
@@ -87,6 +94,12 @@ func (c *multiNamespaceCache) GetInformer(ctx context.Context, obj client.Object
 		}
 		informers[ns] = informer
 	}
+	clusterCacheInf, err := c.clusterCache.GetInformer(ctx, obj)
```

Comment:
I don't think this is correct, this should only happen if the obj is not namespaced

Reply:
Done, modified this to have `clusterCacheInformer` only when object is cluster scoped.